forms.py

```diff
@@ -0,0 +1,7 @@
+from flask_wtf import FlaskForm

```

Comment:
You'll need to add this to requirements.txt for this to works (same for wtforms)

Reply:
Sure. I executed `pip freeze > requirements.txt` and it captured a few more dependencies, but hopefully that is ok.